src/NJsonSchema.Annotations/NJsonSchema.Annotations.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
we should probably go with

<TargetFrameworks>netstandard2.0;net462</TargetFrameworks>